README.md

```diff
@@ -11,6 +11,13 @@ page](https://git-scm.com/docs/git-gui).
 Git GUI was initially written by Shawn O. Pearce, and is distributed with the
 standard Git installation.
 
+Examples of `gitk` and `git citool`, the most popular tools in the git-gui
+suite:
+
+![gitk example](docs/gitk-example.png)
```

Comment:
Strictly speaking `gitk` isn't part of Git GUI. It's a separate project (though both do come bundled with git) and can be called from a menu in Git GUI.

Reply:
OK, replaced `gitk` with a `git gui blame` example now.